src/hydra/schemaAnalyzer.ts

```diff
@@ -29,9 +29,13 @@ const getFieldNameFromSchema = (schema: Resource) => {
  * @returns The type of the field
  */
 const getFieldType = (field: Field) => {
-  switch (field.id) {
+  const withHttpScheme = (value: string | null | undefined) =>
```

Comment:
I think we should make the code uniform: either use this function for `getFieldNameFromSchema` too or use regular expressions here.

Reply:
Done